cfme/tests/automate/test_service_dialog.py

```diff
@@ -1,165 +1,114 @@
 # -*- coding: utf-8 -*-
-import fauxfactory
 import pytest
+import fauxfactory
 
 from cfme import test_requirements
-from cfme.automate.service_dialogs import ServiceDialog
-from utils import error
+from cfme.automate.service_dialogs import DialogCollection
 from utils.update import update
-
-
-pytestmark = [test_requirements.automate, pytest.mark.tier(3)]
-
-
-def test_create_service_dialog():
+from utils import version, error
+
+pytestmark = [
+    pytest.mark.ignore_stream("upstream"),
+    test_requirements.automate,
+    pytest.mark.tier(3)
+]
+
+
+def create_dialog(element_data, label=None):
```

Comment:
This function should also accept appliance and pass it into the DialogCollection instantiation. Then the tests can use the appliance fixture and pass it into create_dialog.